Scripts~/windows/.build-gamelift-server-sdk.ps1

```diff
@@ -59,8 +57,8 @@ else
 }
 
 echo "Building GameLift Server SDK..."
-Invoke-Expression '& $NUGET_EXE_PATH restore "$SDK_PATH\GameLiftServerSDKNet45.sln" -PackagesDirectory "$SDK_PATH\packages"'
-dotnet build "$SDK_PATH\Net45\GameLiftServerSDKNet45.csproj" --configuration Release --output "$RUNTIME_PLUGINS_PATH"
+dotnet restore "$SDK_PATH\GameLiftServerSDK.sln" --packages "$SDK_PATH\packages"
+dotnet build "$SDK_PATH\src\GameLiftServerSDK\GameLiftServerSDK.csproj" --configuration Release -f net462 --output "$RUNTIME_PLUGINS_PATH"
```

Comment:
Added to the list of blockers, but this needs to also be allowed via the standalone plugin. Probably that's a README update, but needs to be done.

Reply:
We should also update the README to reflect that the sample game only works for net462 and not net6